test/e2e/apimachinery/chunking.go

```diff
@@ -122,6 +122,29 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 		gomega.Expect(list.Items).To(gomega.HaveLen(numberOfTotalResources))
 	})
 
+	ginkgo.It("should chunk lists of Pods", func() {
+		ns := f.Namespace.Name
+		c := f.ClientSet
+		client := c.CoreV1().PodTemplates(ns)
+		loopCount := int64(0)
+		for {
+			opts := metav1.ListOptions{}
+			opts.Limit = int64(rand.Int31n(numberOfTotalResources/10) + 1)
+			list, err := client.List(opts)
+			framework.ExpectNoError(err, "failed fetching PodTemplate list")
+
+			if loopCount > 0 && len(list.Continue) > 0 {
+				gomega.Expect(list.Continue[loopCount]).NotTo(gomega.Equal(list.Continue[loopCount-1]), "chunks are the same")
```

Comment:
list.Continue is a string, right?

why are you using loop count as an index into this string?

the intention is to test that continue is different over different iterations of the loop? but that's not a valid assumption if e.g. you're on the last page.

Reply:
I have the same questions.

Reply:
I have updated the test in cee471ad3a972b37fd928aa6af1891d8323b75df to store `list.Continue` in an array for comparison, since it itself isn't an array of `Continue`s.